minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Mixins/MixinMinecraftGameloop.java

```diff
@@ -268,15 +264,4 @@ private void runGameLoop() throws IOException
 
         this.mcProfiler.endSection(); //root
     }
-    
-    @Overwrite
```

Comment:
This change is needed for the re spawn code to work - will create conflicts. We should be sure to use this.leftClickCounter=0 to prevent the agent from getting stuck in the guiGameOver screen

Reply:
Fixed!